README.md

```diff
@@ -1,2 +1,21 @@
-# o11y
-Stonesoup observability 
+# RHTAP Observability 
+
+This repository contains Prometheus recording rule files and alert rule files for observability use, as well as unit tests for these rules and alerts.
+
+## Recording Rules
+
+- `appstudio_container_resource_limits` collects all the resources limits (cpu and memory) for all containers and init_containers, and adding the `label_pipelines_appstudio_openshift_io_type` label to our metric
+
+- `appstudio_container_resource_minutes_gauge` uses the `appstudio_container_resource_limits` to create a 1 minutes gauge and calculate the limits according to the live time of the container
```

Comment:
1 minute (no 's').
Should "live time" be "lifetime" or "live-time" is a common expression?

Reply:
`resources limits` -> `resource limits`

Reply:
Done